Improved writing style, part 1. (#22213)
highcharts/highcharts: docs/datagrid/installation.md

===

## Install via NPM

> One way to add the DataGrid to your web page is via npm script.

Reviewer: ```suggestion
One way to add Highcharts DataGrid to your web page is via npm script.
```

> You can get all of the functionality of the DataGrid through Dashboard packages, which can be installed using the following script:

Reviewer: ```suggestion
You can get all of the functionality of the datagrid through Highcharts Dashboards packages, which can be installed using the following script:
```

```bash
npm install @highcharts/dashboards
```
Then import the package into your project:
``` js
import DataGrid from '@highcharts/dashboards/datagrid';
```

> ## Include DataGrid by script tag

Reviewer: ```suggestion
## Include Highcharts DataGrid by script tag
```

Include the JavaScript files in your web page's `<head>` section, as shown below.

```html
<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
 ```

> In the example above, the Javascript files are loaded from [code.highcharts.com](https://code.highcharts.com), but you can download the files from [highcharts.com](https://www.highcharts.com/download/) and include them on your web page. Here is an example with DataGrid loaded from your server:

Reviewer: ```suggestion
In the example above, the Javascript files are loaded from [code.highcharts.com](https://code.highcharts.com), but you can download the files from [highcharts.com](https://www.highcharts.com/download/) and include them on your web page. Here is an example with Highcharts DataGrid loaded from your server:
```

### Get started

> Now, you are ready to use DataGrid. Check out [Your first DataGrid](/docs/datagrid/your-first-datagrid) to get started.

Reviewer: ```suggestion
Now, you are ready to use Highcharts DataGrid. Check out [Your first Highcharts DataGrid](/docs/datagrid/your-first-datagrid) to get started.
```
